src/main/java/org/skriptlang/skript/scheduler/TaskManager.java

```diff
@@ -0,0 +1,104 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package org.skriptlang.skript.scheduler;
+
+import java.util.concurrent.Callable;
+import java.util.concurrent.Future;
+import java.util.concurrent.TimeUnit;
+
+import org.skriptlang.skript.scheduler.platforms.FoliaScheduler;
+import org.skriptlang.skript.scheduler.platforms.SpigotScheduler;
+
+import ch.njol.skript.Skript;
+
+/**
+ * Class to handle the abstract versioning and to be loaded from the Skript main class.
```

Comment:
The comment doesn't make much sense. I would suggest instead maybe `Handles the scheduling of tasks in a platform-agnostic manner. Loaded by the Skript main class.`